tests/features/custom_page/api.feature

```diff
@@ -6,15 +6,16 @@ Feature: Custom page API
 
   Scenario: Programmatically create a Custom page
     Given the following collection:
-      | title             | Le Foie Heureux |
-      | owner             | Rufus Drumknott |
-      | logo              | logo.png        |
-      | moderation        | yes             |
-      | closed            | yes             |
-      | elibrary creation | facilitators    |
+      | uri               | http://joinup.eu/custom_page_api_collection |
+      | title             | Le Foie Heureux                             |
+      | owner             | Rufus Drumknott                             |
+      | logo              | logo.png                                    |
+      | moderation        | yes                                         |
+      | closed            | yes                                         |
+      | elibrary creation | facilitators                                |
     And custom_page content:
-      | title      | body                                     | groups audience |
-      | Dummy page | This is some dummy content like foo:bar. | Le Foie Heureux |
+      | title      | body                                     | groups audience                             |
+      | Dummy page | This is some dummy content like foo:bar. | http://joinup.eu/custom_page_api_collection |
```

Comment:
The problem I found out is that never actually worked :/ I tried to see this in the UI and there wasn't any connection being created. And due to the fact that these are api.feature tests, we were not seeing the problem. But then other tests started to break, like the behat scenario test.

For the rdf entities, we were having our own create functionality and we were manually loading the object. But the node is being handled by Drupal context directly.

Though, Francesco had an idea with a hook and I am going to look into that.